task/ecosystem-cert-preflight-checks/OWNERS

```diff
@@ -1,2 +1,5 @@
+approvers:
+  - skattoju
```

Comment:
I think you should add another devs that are mentioned in the comment

Reply:
Updated now. Thanks.